src/test/java/io/nats/client/NatsTestServer.java

```diff
@@ -17,8 +17,13 @@
 import nats.io.NatsServerRunner;
 
 import java.io.IOException;
+import java.util.logging.Level;
 
 public class NatsTestServer extends NatsServerRunner {
+    static {
+        NatsServerRunner.setLoggingLevel(Level.SEVERE);
+    }

```

Comment:
Also on purpose (just checking again, can imagine it being useful for local testing)

Reply:
Yes. The unit tests spew out all the server info and it makes debugging test harder. This removes that.